healpy/src/_query_disc.pyx

```diff
@@ -344,8 +344,11 @@ cdef pixset_to_array(rangeset[int64] &pixset, buff=None):
             ii += 1
     return ipix
 
-cdef bool isnsideok(int nside):
-     return nside > 0 and ((nside & (nside -1))==0)
+cdef bool isnsideok(int nside, bool nest):
```

Comment:
what about having `nest` default to False?
That is common in all other functions that accept that argument.